rollup.config.js

```diff
@@ -128,23 +119,14 @@ export default [
 			json(),
 			ts_plugin
 		],
-		output: [
-			{
-				file: 'compiler.js',
-				format: is_publish ? 'umd' : 'cjs',
-				name: 'svelte',
-				sourcemap: true,
-			},
-			{
-				file: 'compiler.mjs',
-				format: 'esm',
-				name: 'svelte',
-				sourcemap: true,
-			}
-		],
+		output: {
+			file: 'compiler.cjs',
+			format: is_publish ? 'umd' : 'cjs',
+			name: 'svelte',
+			sourcemap: false
```

Comment:
Shouldn't we still have sourcemaps for this?

Reply:
We decided to remove it because rarely any tool depends on it, and those who do can still backtrack manually